api/tests/services/training/test_extraction_filters.py

```
@@ -0,0 +1,121 @@
+
+    BISQ_TERMS = {
+        "bisq",
+        "spv",
+        "mediation",
+        "ctrl+o",
+        "seed",
+        "wallet",
+        "trade",
+        "offer",
+        "security deposit",
+    }
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)